sql_generators/glean_usage/templates/event_monitoring_live.view.sql

```diff
@@ -1,8 +1,31 @@
-CREATE OR REPLACE VIEW `{{ project_id }}.{{ target_view }}` AS
-{% for app in apps %}
-{% set outer_loop = loop -%}
-{% for dataset in app -%}
-{% if dataset['bq_dataset_family'] in prod_datasets %}
+CREATE OR REPLACE VIEW
+  `{{ project_id }}.{{ target_view }}` AS
+  {% for app in apps %}
+    {% set outer_loop = loop -%}
+    {% for dataset in app -%}
+      {% if dataset['bq_dataset_family'] in prod_datasets
+        and dataset['bq_dataset_family'] in event_tables_per_dataset %}
```

Comment:
This line is the actual change, the rest is `bqetl format`.  Hiding whitespace makes it easier to read